app/controllers/album.controller.js

```diff
@@ -116,4 +117,35 @@ exports.delete = (req, res) => {
         message: "Could not delete Album with id=" + id
       });
     });
-};
\ No newline at end of file
+};
+
+exports.getAlbumTracks =  async (req, res) => {
+  const id = req.params.albumId  
+  const data = await Album.findOne({
+      include: [{
+          model: Track,
+          as: 'tracks'
+      }],
+      where: { id: id }
+  });
+  res.status(200).send(data)
+}
+
+// exports.findByTitle = (req, res) => {
```

Comment:
Whats the plan with the "find by title" method? I see that it is commented out.

Reply:
I have all these functions since Friday, and I know I have to create one PR at a time, so I was leazy creating branches all the times

Reply:
Cool!